src/js/tools/ModulesCollector.ts

```diff
@@ -1,7 +1,9 @@
-/* eslint-disable no-console */
+import { logger } from '@sentry/utils';
 import { existsSync, mkdirSync, readFileSync, writeFileSync } from 'fs';
 import { posix, sep } from 'path';
 
+logger.enable();
```

Comment:
M: Is this needed? I don't see this anywhere else.

Reply:
For RN SDK it's hidden in [`initAndBind`](https://github.com/getsentry/sentry-javascript/blob/8570f0d29567e5f7cebf9a18f82e019191f2ebee/packages/core/src/sdk.ts#L22)
But as this run as a standalone script, I needed to enable the logger.